OpenRA.Mods.Common/Traits/AutoTarget.cs

```diff
@@ -345,6 +345,8 @@ public bool HasValidTargetPriority(Actor self, Player owner, BitSet<TargetableTy
 			});
 		}
 
+		readonly List<AutoTargetPriorityInfo> validPriorities = [];
```

Comment:
Maybe declare this inside of ChooseTarget just so it's a bit more local to usage, but say outside the `foreach (var target in targetsInRange)` loop.

Reply:
ChooseTarget is called very often, why shouldn't we just use class variables for perf?

Reply:
ok